### ISSUE TITLE
tempearate literals not passing step 47 build a calorie counter

### ISSUE THREAD

**Issue by blindgeekzone** [freeCodeCamp/freeCodeCamp#62340]
### Describe the Issue

hi, using the jaws for windows screen reader totally blind, have tried on other browsers, but up to step 48. and now it will not pass. so to then add a literal to a html label and then have the text of Entry number on the label for this project. it is not passing, dont have any trailling spaces at the start or the end of the line of code, so do this code:
```
const HTMLString =  `<label>Entry${entryNumber} Name</label> `;
```

and it says i need to have the label in the literal and is not passing, have researched on google and the code almost looks like the same. so you have a issue with your tests or vallidator. so, need to fix urgenbtly, and it is not picking up the tests when i am in chrome or edge and says 0 out of 214 steps when i am logged in, and in firefox, does not says the number of the tests, just displays the browser preview. i know this is a beta curriculum, when are you going to role out the stable curricujlum and also fix issues with the tests and the online vallidator. and also have a built in way to delete left over cache or left over files, i am only one blind person doing this course, but have encountered barriers, but used google and also the forum to then get most of the projects done using a screen reader. cannot see and so maybe hidden characters. please fix this, if as a blind developer i am having issues, wondering if sighted students are having the same issue and encountering the same fcc problems, please fix urgent. maybe quincy can then take a look and doing the project out of 97 steps and up to step 48, 3 are failling to pass as fcc is very literal and strict, no flexability..  

### Affected Page

https://www.freecodecamp.org/learn/full-stack-developer/workshop-calorie-counter/step-47

### Steps to Reproduce

hi, okay type the code or i do have a backup local copy in visual studio code and then either type into the editor or in vs code and then copy and paste to the editor. then i hear not passed and does not give me the tests and just says i should add the label inside the literal variable in javascript.the 1. Go to '...'
2. Click on '...'
3. Scroll down to '...'

### Expected behavior

that it should pass the testptests and then pass the step, or else fix the issue of not reading me the tests and the numbers, was doing it earlier today, but not at all and firefox is the only brower which sort of works with fcc, did try chrome, firefox, which is stable, and edge and even installed brave. please fix this error or flaws or maybe role out the stable curriculum and not the beta one.   

### System

windows 11 pro 2024- Device: [e.g. iPhone 6, Laptop]
- OS: [e.g. iOS 14, Windows 10, Ubuntu 20.04]
- Browser: [e.g. Chrome, Safari]
- Version: [e.g. 22]

### Additional context

hi. a blind screen reader user, so then have to move on to the next step and then wait for a resolution of fcc falling over or critical flaws with the online curriculum online editor. if you are going to have beta versions, then fix the errors and role out the official stable version, maybe quincy can fix this it is his baby.

**Comment by jeremylt** [comment]
The code you have provided is not a valid solution. Please make a post on the forum requesting help before creating an issue stating the Step is broken. Thanks
